airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py

```diff
@@ -46,10 +46,12 @@ class Config:
 
     environment: Union[ProductionEnvSpec, SandboxEnvSpec] = Field(default=ProductionEnvSpec.Config.title)
 
-    access_token: str = Field(description="Long-term Authorized Access Token.", airbyte_secret=True)
+    access_token: str = Field(description="The Long-term Authorized Access Token.", airbyte_secret=True)
```

Comment:
do these have a default `title` property set?

Reply:
Yes, on the UI all titles are set correctly.